### PR TITLE
fix(webapp): use strict equality

### PR DESCRIPTION
**Pull request by Rotzbua**
Use strict equality for type safety.

### CODE REVIEW

File: www/webapp/src/components/Field/Record/DNSKEY.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@ const int8 = between(0, MAX8);
 const MAX16 = 65535;
 const int16 = between(0, MAX16);
 
-const equals3 = (value) => !value || value == 3;
+const equals3 = (value) => !value || value === 3;
```
Comment (peterthomassen) [inline www/webapp/src/components/Field/Record/DNSKEY.vue:13]: Is this correct? -- The input field value is expected to be a string, so `=== 3` should always be false.

**Review by peterthomassen — CHANGES_REQUESTED** [review]
I think one of the changes causes problems (see comment at the respective code location).

All other changes are about comparisons where operand types are not under user control (e.g. the `.length` attribute of an array), so type safety is implicit because the operand is type-safe.

While I see that this change could be made, I'm not sure this is worth merging. -- I'm happy to be educated about the actual impact, or about a recent development where JavaScript folks feel that these changes are important.

Also, @nils-wisiol what do you think?

**Comment by Rotzbua** [comment]
Thanks for the feedback. I reduced the changes to just type `String`. As you pointed out the compare with integer need way more investigation.

**Comment by nils-wisiol** [comment]
Thanks for this work @Rotzbua! 

I believe less linter warnings are good, also because we do not have any automated tests for the webapp. Maintaining good code style will (a) produce less error-prone code and (b) unclog the linter output to see things that actually matter.

**Comment by nils-wisiol** [comment]
Is there a way to integrate the linter in out GitHub actions? I do not think we can require zero warnings as of now. So ideally, PRs would be required to not produce more warnings than the code in main :)

**Review by nils-wisiol — APPROVED** [review]

**Comment by nils-wisiol** [comment]
@peterthomassen this needs your approval for merging

**Review by peterthomassen — APPROVED** [review]

**Comment by Rotzbua** [comment]
@nils-wisiol Unfortunately I don't know any tool which fulfills this. Otherwise I would have already created a PR ;)

**Pull request merged**